app/styles/vendor/_material.scss

```diff
@@ -8,9 +8,11 @@ $radio-color: $brand-info;
 $input-text-highlight-color: $brand-info;
 $input-text-bottom-border-color: $gray;
 $input-text-label-color: $gray-darker;
-$input-text-padding: 8px;
+$input-text-font-size: 13px;
```

Comment:
Can we use font-base-size? Same at line 15

Reply:
Right now we don't have a variable file that is shared between mdl and bootstrap. Once the scss restructure is done, I will add a common variable file. That change will come in future as it is dependent on other changes.